.github/workflows/update-test-image.yml

```diff
@@ -0,0 +1,39 @@
+name: update-test-image
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Get Version
+        id: get-version
+        run: yq -r '"version=\(.version)"' source/charts/k8s-monitoring/Chart.yaml >> $GITHUB_OUTPUT
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/grafana/k8s-monitoring-test
+          tags: |
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
```

Comment:
Yes, this will always be true, but I'd rather leave this as a breadcrumb if we ever expand this to build images based on PRs.